typescript/infra/config/environments/testnet3/helloworld.ts

```diff
@@ -12,7 +12,7 @@ export const hyperlane: HelloWorldConfig<TestnetChains> = {
   kathy: {
     docker: {
       repo: 'gcr.io/abacus-labs-dev/hyperlane-monorepo',
-      tag: 'sha-6ee34e4',
```

Comment:
Was this intentional? What changes did we have in kathy?

Reply:
More than anything just wanted to get the new tag in place. I don't think there were any significant changes.